server/package.json

```diff
@@ -6,11 +6,13 @@
   "private": true,
   "scripts": {
     "dev": "nodemon -w src --exec \"babel-node src\"",
+    "dev-test": "NODE_ENV=test PORT=8000 babel-node src",
```

Comment:
Here is a dev testing server. 

`PORT=8000` is a code smell and should be changed to dotenv